lib/logging/logging.dart

```diff
@@ -110,11 +115,16 @@ class LoggingScreen extends Screen {
 
     // Log GC events.
     service.onGCEvent.listen((Event e) {
-      final dynamic json = e.json;
-      final String message = 'gc reason: ${json['reason']}\n'
-          'new: ${json['new']}\n'
-          'old: ${json['old']}\n';
-      _log(new LogData('gc', message, e.timestamp));
+      final String summary =
+          '${e.json['reason']} collection, ${e.json['isolate']['name']}';
+      final Map<String, dynamic> event = <String, dynamic>{
+        'reason': e.json['reason'],
+        'new': e.json['new'],
+        'old': e.json['old'],
+        'isolate': e.json['isolate'],
+      };
+      final String message = jsonEncode(event);
+      _log(new LogData('gc', message, e.timestamp, summary: summary));
```

Comment:
Is there any value in having `LogData` hold the JSON (here we decode it and re-encode it, and then in the display we repeat the same)?

Reply:
We don't currently have access to it, but I do agree that there's no need to decode and re-encode; will see what we can clean up here.